cas-server-support-ldap/src/main/java/org/jasig/cas/authentication/LdapAuthenticationHandler.java

```diff
@@ -152,7 +153,7 @@ protected HandlerResult authenticateUsernamePasswordInternal(final UsernamePassw
         final AuthenticationResponse response;
         try {
             logger.debug("Attempting LDAP authentication for {}", upc);
-            final String password = getPasswordEncoder().encode(upc.getPassword());
+            final String password = Strings.nullToEmpty(getPasswordEncoder().encode(upc.getPassword()));
```

Comment:
it starts off as null in the `UsernamePasswordCredential`, somehow it isn't being set. I'll try to diagnosis a bit more to see if I'm doing something unique in my deployment that is causing an odd state.